scene/main/viewport.cpp

```diff
@@ -3033,6 +3093,15 @@ void Viewport::pass_mouse_focus_to(Viewport *p_viewport, Control *p_control) {
 	}
 }
 
+void Viewport::_validate_property(PropertyInfo &property) const {
+	if (property.name == "fsr_upscale_quality" || property.name == "fsr_upscale_sharpness" || property.name == "fsr_upscale_custom_quality" || property.name == "fsr_upscale_mipmap_bias") {
+		bool is_mobile = (bool)GLOBAL_GET("rendering/vulkan/rendering/back_end");
```

Comment:
Is there no better way to check if this is the mobile renderer? This might work for now, but will break if it's not updated when the mobile gles3 (or any other) renderer is added.

Reply:
Changed `is_mobile` check to use `RSG::rasterizer->is_low_end()`